crypto/ake/enclave/src/lib.rs

```diff
@@ -84,7 +84,7 @@ pub struct AkeEnclaveState<EI: EnclaveIdentity> {
     ias_pending: Mutex<LruCache<IasNonce, Quote>>,
 
     /// The cached attestation evidence, if any.
-    current_attestation_evidence: Mutex<Option<VerificationReport>>,
+    current_attestation_evidence: Mutex<Option<EvidenceKind>>,
```

Comment:
Is this a similar situation to https://github.com/mobilecoinfoundation/mobilecoin/pull/3602/files#r1353583390 where we actually know there is only one type ever being used in a given build?